frame/clients/maximizing_gtk_app.py

```diff
@@ -0,0 +1,22 @@
+import gi
+try:
+    gi.require_version("Gtk", "4.0")
+except ValueError:
+    gi.require_version("Gtk", "3.0")
```

Comment:
no no, 3 and 4 are not API compatible so even if this does work now, it would require us to test both whenever we change the code in this file. There's also a chance 3 and 4 will behave differently in some way significant to the test. The two libraries can be installed side-by-side, so I see no reason why we don't chose the one we want and make it a dependency of the project.

Reply:
Then I messed something up - needed 3.0 on jammy, but it didn't work for me on kinetic…

Reply:
`libgtk-4-dev` is available on jammy and up, it might be as simple as installing that?